superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/buildQuery.ts

```diff
@@ -75,6 +82,7 @@ export default function buildQuery(formData: QueryFormData) {
         time_offsets: isTimeComparison(fd, queryObject) ? fd.time_compare : [],
         post_processing: [
           pivotOperatorInRuntime,
+          sortOperator(fd, queryObject),
```

Comment:
The sortOperator should move before the flattenOperator because the other operators(rolling, resample, and so on) also changed original data.